File: baekjoon/shim/Java/단계별로_풀어보기/기본_수학_2/n_2581.java
package Study_2022.baekjoon.shim.Java.단계별로_풀어보기.기본_수학_2;

import java.util.*;

public class n_2581 {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int n1=sc.nextInt(), n2=sc.nextInt(), sum=0, min=0;
        for(int i=n1; i<=n2; i++){
            int count=0;
            for(int j=2; j<=Math.sqrt(i); j++){
                if(i%j==0) count++;
            }
            if(count==0&&i!=1){
                if(min==0)min=i;
                sum+=i;
            }
        }
        System.out.println(sum==0?-1:(sum+"\n"+min));
    }
}

File: baekjoon/shim/Python/단계별로_풀어보기/기본_수학_2/n_2581.py
import math
n1=int(input())
n2=int(input())
sum=0
min=0
for i in range(n1, n2+1):
    if i==1: continue
    count=0
    for j in range(2,int(math.sqrt(i))+1):
        if i%j==0:
            count+=1
    if(count==0):
        if min==0: min=i
        sum+=i
print(-1 if sum==0 else (f'{sum}\n{min}'))